<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="4" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="63756" started-at="2020-11-18T09:48:28 CST" finished-at="2020-11-18T09:49:32 CST">
    <groups>
    </groups>
    <test name="Test1" duration-ms="63756" started-at="2020-11-18T09:48:28 CST" finished-at="2020-11-18T09:49:32 CST">
      <class name="com.test.demo.SmokeTesting">
        <test-method status="PASS" signature="doBeforeClass()[pri:0, instance:com.test.demo.SmokeTesting@50c87b21]" name="doBeforeClass" is-config="true" duration-ms="4727" started-at="2020-11-18T09:48:28 CST" finished-at="2020-11-18T09:48:33 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doBeforeClass -->
        <test-method status="PASS" signature="testLogin()[pri:0, instance:com.test.demo.SmokeTesting@50c87b21]" name="testLogin" is-config="true" duration-ms="11771" started-at="2020-11-18T09:48:33 CST" finished-at="2020-11-18T09:48:45 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogin -->
        <test-method status="PASS" signature="testLogoOff()[pri:1, instance:com.test.demo.SmokeTesting@50c87b21]" name="testLogoOff" duration-ms="15595" started-at="2020-11-18T09:48:45 CST" finished-at="2020-11-18T09:49:00 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogoOff -->
        <test-method status="PASS" signature="testLogoOn()[pri:2, instance:com.test.demo.SmokeTesting@50c87b21]" name="testLogoOn" duration-ms="15456" started-at="2020-11-18T09:49:00 CST" finished-at="2020-11-18T09:49:16 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogoOn -->
        <test-method status="PASS" signature="testContainerOff()[pri:5, instance:com.test.demo.SmokeTesting@50c87b21]" name="testContainerOff" duration-ms="10" started-at="2020-11-18T09:49:16 CST" finished-at="2020-11-18T09:49:16 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testContainerOff -->
        <test-method status="FAIL" signature="testContainerOn()[pri:6, instance:com.test.demo.SmokeTesting@50c87b21]" name="testContainerOn" duration-ms="15342" started-at="2020-11-18T09:49:16 CST" finished-at="2020-11-18T09:49:31 CST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.test.demo.SmokeTesting.testContainerOn(Unknown Source)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.TestNG.privateMain(TestNG.java:1336)
at org.testng.TestNG.main(TestNG.java:1304)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testContainerOn -->
        <test-method status="PASS" signature="doAfterMethod()[pri:0, instance:com.test.demo.SmokeTesting@50c87b21]" name="doAfterMethod" is-config="true" duration-ms="835" started-at="2020-11-18T09:49:31 CST" finished-at="2020-11-18T09:49:32 CST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- doAfterMethod -->
      </class> <!-- com.test.demo.SmokeTesting -->
    </test> <!-- Test1 -->
  </suite> <!-- Suite -->
</testng-results>
